docs/resources/static_route.md

```diff
@@ -0,0 +1,59 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "maas_static_route Resource - terraform-provider-maas"
+subcategory: ""
+description: |-
+  Provides a resource to manage MAAS static routes.
+---
+
+# maas_static_route (Resource)
+
+Provides a resource to manage MAAS static routes.
+
+## Example Usage
+
+```terraform
+resource "maas_subnet" "tf_subnet" {
+  cidr       = "10.88.88.0/24"
+  fabric     = maas_fabric.tf_fabric.id
+  vlan       = maas_vlan.tf_vlan.vid
+  name       = "tf_subnet"
+  gateway_ip = "10.88.88.1"
+  dns_servers = [
+    "1.1.1.1",
+  ]
+}
+
+resource "maas_static_route" "tf_static_route" {
+  source = maas_subnet.tf_subnet.name
+  destination = "172.18.0.0/16"
```

Comment:
The schema says this is the `Destination subnet name for the route.`, as this is the name of a second subnet you're linking to the source, but you've provided a CIDR here instead?